File: src/ProjectR.Scripting/content/scripts/spells/SharingIsCaring.lua
function GetTargetType()
    return Single
end

function GetName()
  return "Sharing is caring"
end

function GetDescription()
    return "Imbue your target with the power of the wind.\nLow delay.\nIncreases SPD."
end

function IsSupportSpell()
    return true
end

function GetMPCost()
    return .28
end

function GetDelay()
  return .75
end

function GetSpellType()
  return Magical
end

function GetMasteries()
  return {  }
end

function SpellEffect()
  d:BuffStat(SPD, .5)
end

File: src/ProjectR.Scripting/content/scripts/spells/SharingIsCaring.cs
using System.Collections.Generic;
using ProjectR.Interfaces;
using ProjectR.Interfaces.Model;
using ProjectR.Interfaces.Model.Stats;

namespace ProjectR.Scripting
{
    public class SharingIsCaring : SpellScriptBase
    {
        public override string Name { get { return "Sharing is caring"; } }
        public override string Description { get { return "Imbue your target with the power of the wind.\nLow delay.\nIncreases SPD."; } }

        public override TargetType TargetType { get { return TargetType.Single; } }
        public override IList<EleMastery> Masteries { get { return new EleMastery[] { }; } }
        public override SpellType SpellType { get { return SpellType.Magical; } }
        public override bool IsSupportSpell { get { return true; } }

        public override double MPCost { get { return 28; } }
        public override double Delay { get { return .75; } }

        public override void SpellEffect(ICharacter caster, ICharacter target)
        {
            BuffStat(Stat.SPD, .5);
        }
    }
}